_posts/2020-11-09-the-cloud-data-warehouse.md

```diff
@@ -0,0 +1,158 @@
+
+Modern Cloud Data Warehouses can support the existing status quo, including various data modelling methodologies such as Kimball, Inmon or Data Vault for example. Additionally, sticking the word ***cloud*** in front of *data warehouse* does what you would expect cloud to do, in addressing many of the technological limitations or complications associated with traditional Data Warehouses - namely storage availability, compute scalability, and high availability.
+
+However, modern Cloud Data Warehouse offerings typically go far beyond that. Whilst delivering the core competencies of what a Data Warehouse should offer, they often extend these capabilities with features not readily seen before. For example:
+
+* [BigQuery ML](https://cloud.google.com/bigquery-ml/docs/introduction) allows users to inference with Machine Learning models against their datasets directly, using SQL. A more generic feature is also available on Snowflake using [External Functions](https://docs.snowflake.com/en/sql-reference/external-functions-introduction.html) 
+* Snowflake's [Zero Copy Data Sharing](https://docs.snowflake.com/en/user-guide-data-share.html) across accounts in the same cloud region, and [Replication Across Cloud Platforms](https://docs.snowflake.com/en/user-guide/database-replication-failover.html) as described in our recent blog post on [Snowflake Organizations](https://mechanicalrock.github.io/2020/09/23/snowflake-org-structures.html), are a powerful enabler for complex organisations, especially those navigating multi cloud strategies
+* Data Time Travel features of both [Snowflake](https://docs.snowflake.com/en/user-guide/data-time-travel.html) and [BigQuery](https://cloud.google.com/bigquery/docs/time-travel), allowing you to see data as it was at a particular point in time
+
+As many organisations evolve to consist of more multi-disciplinary teams, so too has technology adapted to more simply support the kind of collaboration and critical insight work that most organisations require. And so, a *modernised view* of the Data Warehouse is upon us where the technology is no longer simply a keystone of the architecture, but rather a platform for enhancement and enablement of data.
+
+### Resilient Architecture
+
+One of the most understated advantages of Cloud Data Warehouses is the resiliency and availability. To be clear, we are talking about service reliability, over data reliability (more on that in the next post). Many large organisations who have presence in multiple data centres for availability concerns are beginning to transition their secondary data centres to be cloud based. Whilst there can be significant operational and capital expenditure savings from this, it often leaves organisations with old world problems:
```

Comment:
"... Data Warehouses is the resiliency and availability..."   = "is their resiliency" ????

Reply:
maybe needs a rewording. does the resiliency and availability refer to the "advantage" or to the "cloud data warehouse"